iib/workers/tasks/build_merge_index_image.py

```diff
@@ -244,11 +245,17 @@ def handle_merge_request(
         deprecation_bundles = deprecation_bundles + [
             bundle['bundlePath'] for bundle in invalid_version_bundles
         ]
-        intermediate_image_name = _get_external_arch_pull_spec(
-            request_id, arch, include_transport=False
-        )
 
         if deprecation_bundles:
+            # opm can only deprecate a bundle image on an existing index image. Build and
```

Comment:
could you add the comment that the index image has to be pushed to the registry and we CAN'T use it from internal registry?

Reply:
added